review-tools/pick-to-branch

```diff
@@ -69,8 +69,18 @@ m*)
     ;;
 esac
 
+REMOTE=`git branch --list $TARGET -vv | sed 's/^\* //' \
+ | awk '{ sub(/\[/, "", $3); sub(/\/.*/, "", $3);  print $3 }'`
+# usually this will be 'upstream'
+if [ "$REMOTE" = "" ] ; then
+    echo Cannot find git remote for target branch $TARGET
+    exit 1
+fi
+git remote get-url $REMOTE | grep -q -E 'github.openssl.org:(openssl|omc|otc)' \
```

Comment:
Please drop the `(openssl|omc|otc)` part from the regexp. No need to be too restrictive.

Reply:
Done